packages/app-store/_components/DynamicComponent.tsx

```diff
@@ -1,9 +1,17 @@
-export function DynamicComponent<T extends Record<string, any>>(props: { componentMap: T; slug: string }) {
+export function DynamicComponent<T extends Record<string, any>>(props: {
+  componentMap: T;
+  slug: string;
+  wrapperClassName?: string;
+}) {
   const { componentMap, slug, ...rest } = props;
 
   if (!componentMap[slug]) return null;
 
   const Component = componentMap[slug];
 
-  return <Component {...rest} />;
+  return (
+    <div className={props.wrapperClassName || ""}>
```

Comment:
Adds the className support through wrapper so that the app's AppSettings component never requires to read className